channel/kustomization.yaml

```diff
@@ -1,3 +1,4 @@
+---
 resources:
-- channel.yaml
-- namespace.yaml
+  - channel.yaml
```

Comment:
I am not sure we need this indent for lists, from what I can [see](https://yaml.org/spec/1.2.2/#21-collections) we should actually avoid it. Is there a reason this is indented here and in other files where lists are in use?

Reply:
We need this to keep yamllint happy.

Reply:
k8s outputs sequences with indentation of the parent node, controller-sdk does the same etc. I am "thinking" we modify the yamllint to add the [rule](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation) for `indent-sequences`.

Reply:
Default yamllint rules creates more readable yaml.